orchagent/intfsorch.cpp

```diff
@@ -86,7 +86,16 @@ void IntfsOrch::doTask(Consumer &consumer)
         {
             if (alias == "lo")
             {
-                addIp2MeRoute(ip_prefix);
+                // set request for lo may come after warm start restore
```

Comment:
Does it related to warm start? If not, let's do it in another PR.

Reply:
It is related to warm start.  It is noticed that sometime IP data for lo comes after warm start.

Reply:
The logic is also applied to cold start. Are you fixing a bug also exists in cold start? if yes, we prefer a separate PR just for it.

---
In reply to: [209787586](https://github.com/Azure/sonic-swss/pull/554#discussion_r209787586) [](ancestors = 209787586)